apps/lcr/src/store/lcr.ts

```diff
@@ -0,0 +1,33 @@
+// stores/counter.js
+import { defineStore } from 'pinia';
+import axios from 'axios';
+import { PagedMaximoUsers, MaximoUser } from '../types/pagedMaximoUsers';
+
+export const useLcrStore = defineStore('lcr', {
+  state: () => ({
+    pagedMaximoUsers: [],
+  }),
+  getters: {
+    getMaximoUsers(state) {
+      return state.pagedMaximoUsers;
+    },
+  },
+
+  actions: {
+    async fetchMaximoUsers(lastName: string | null) {
+      let url =
+        'https://localhost:7110/api/workstation/GetPaginatedMaximoUsers?pageNumber=1&pageSize=20';
+
+      if (lastName) {
+        url = url + '&lastName=' + lastName;
+      }
```

Comment:
Think about replacing the host with an environment variable. Also, I will recommend wrapping this as a URL and using the searchParams property. I worry about encoding strings for URL safety. See: https://developer.mozilla.org/en-US/docs/Web/API/URL#usage_notes